src/UI/Settings/NetImport/Options/NetImportOptionsView.js

```diff
@@ -20,7 +16,11 @@ var view = Marionette.ItemView.extend({
 	},
 	
 	onShow : function() {
+		var params = new URLSearchParams(window.location.search.slice(1));
```

Comment:
I committed a change that uses a polyfill: https://github.com/WebReflection/url-search-params as recomended by GOOGLE at the bottom of this page: https://developers.google.com/web/updates/2016/01/urlsearchparams?hl=en

I tested in IE.. It didnt work before the change, but works afterwards - could you please test in Safari...